adafruit_imageload/bmp/indexed.py

```diff
@@ -47,7 +47,9 @@ def load(file, width, height, data_start, colors, color_depth, *, bitmap=None, p
 
         file.seek(data_start - colors * 4)
         for value in range(colors):
-            palette[value] = file.read(4)
+            c_bytes = file.read(4)
+            # Need to swap red & green bytes (bytes 0 and 2)
```

Comment:
I think this is actually swapping red and blue.

Reply:
You are correct - I've updated the comment